buildscripts/generate-manifests.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+# Copyright 2020 The OpenEBS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#set -o errexit
+set -o nounset
+set -o pipefail
+
+## find or download controller-gen
+CONTROLLER_GEN=$(which controller-gen)
+
+if [ "$CONTROLLER_GEN" = "" ]
+then
+  echo "ERROR: failed to get controller-gen, Please run make bootstrap to install it";
+  exit 1;
+fi
+
+SCRIPT_ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+
+$CONTROLLER_GEN crd:trivialVersions=true,preserveUnknownFields=false paths=${SCRIPT_ROOT}/pkg/apis/openebs.io/zfs/v1alpha1 output:crd:artifacts:config=deploy/yamls
+
+## create the the crd yamls
+
+echo '
+
+##############################################
+###########                       ############
+###########   ZFSVolume CRD       ############
+###########                       ############
+##############################################
+
+# ZFSVolume CRD is autogenerated via `make manifests` command.
```

Comment:
Probably a note saying "DO NOT MODIFY THIS FILE" can be the first line before this description on how this file can be updated.

Reply:
same comment applies to all auto generated yamls.

Reply:
this line will go the zfs-operator file also. Might confuse the user who is usng it. Feeling better to not put that line?